ironfish/src/node.ts

```diff
@@ -215,12 +215,12 @@ export class IronfishNode {
     })
   }
 
-  async openDB(): Promise<void> {
+  async openDB(options?: { upgrade?: boolean }): Promise<void> {
```

Comment:
Should the `start` command be passing in `{ upgrade: true }`? Or maybe that should be the default?

Reply:
Yes, I made the default true now.

Reply:
Reset might need to pass in `upgrade: false` now